pkg/util/defaults/defaults.go

```diff
@@ -26,7 +26,7 @@ const (
 	Version = "2.0.0-SNAPSHOT"
 
 	// DefaultRuntimeVersion --
-	DefaultRuntimeVersion = "3.20.1-SNAPSHOT"
+	DefaultRuntimeVersion = "2.16.0-SNAPSHOT"
```

Comment:
This one should be already changed in `main` branch. Probably rebasing would sort the same effect.

Reply:
`main` branch is still on 3.20.1. Maybe there is a problem in a script ?

Reply:
Yes, probably. There should be a nightly action that is in charge to regenerate this file.

Reply:
Will be fixed by #4313. You can however remove this change, because when you `make build` it will take care to refresh on its own.

Reply:
I removed the commit :+1:

Reply:
Cool. I've merged the aforementioned PR, at next automatic nightly update, it will refresh everything.